src/components/Header/style.ts

```diff
@@ -64,12 +65,12 @@ const useStyles = makeStyles<Theme, { connected: boolean }>((theme: Theme) => ({
   }),
   link: {
     textDecoration: 'none',
-    minWidth: 'fit-content'
+    minWidth: 'fit-content',
```

Comment:
fix trunk issues

Reply:
done

Reply:
no, there is still a lot of lint errors, fix them